### PR TITLE
Swarm non root

### PR DESCRIPTION
**Pull request by areed**
Run Replicated Swarm services as replicated:docker by id. If the docker group ID is different on a remote node, operator on that node will crash attempting to mount the docker socket. It doesn't matter whether the remote node has a user with the same id as the replicated user created on the master.

### CODE REVIEW

File: install_scripts/templates/swarm/init.sh [hunk 1]
```diff
@@ -25,6 +25,7 @@ NO_CE_ON_EE="{{ no_ce_on_ee }}"
 HARD_FAIL_ON_LOOPBACK="{{ hard_fail_on_loopback }}"
 HARD_FAIL_ON_FIREWALLD="{{ hard_fail_on_firewalld }}"
 ADDITIONAL_NO_PROXY=
+REPLICATED_USERNAME=replicated
```
Comment (areed) [inline install_scripts/templates/swarm/init.sh]: Used by maybeCreateReplicatedUser

Comment (emosbaugh) [inline install_scripts/templates/swarm/init.sh]: why not like this? https://github.com/replicatedhq/replicated-installer/blob/76f9d7d3b9feb2395ec33f3d5de2b009eae2cf56/install_scripts/templates/replicated-operator.sh#L22

Comment (emosbaugh) [reply install_scripts/templates/swarm/init.sh]: how come you dont have to add the replicated user to the worker join ?

Comment (areed) [reply install_scripts/templates/swarm/init.sh]: Updated both. The missing `{{ replicated_username }}` would have broken old installs by trying to run them as root without the migrations in the daemon.

**Pull request merged**